package.json

```diff
@@ -30,8 +30,9 @@
     "test:backend": "go test github.com/kubernetes/dashboard/src/app/backend/...",
     "test:backend:coverage": "./aio/scripts/coverage-backend.sh",
     "test:coverage": "npm run test:frontend:coverage && npm run test:backend:coverage",
-    "test:e2e": "ng e2e",
-    "cluster": "./aio/scripts/start-cluster.sh",
+    "test:e2e": "npm run cluster:start && ng e2e && npm run cluster:stop",
```

Comment:
Not sure if we should start the test cluster by default. What if I would like to test against some other cluster? @floreks WDYT?

Reply:
Yes, this should stay as it was. Usually, we already have a cluster started while developing.

Reply:
Sounds good!